CMakeLists.txt

```diff
@@ -277,7 +278,9 @@ file(GLOB RUNTIME_SRCS
 
 if(BUILD_FOR_HEXAGON)
   # Add file implementing posix_memalign.
-  list(APPEND RUNTIME_SRCS src/runtime/hexagon/hexagon_posix.cc)
+  if(NOT BUILD_STATIC_RUNTIME)
+    list(APPEND RUNTIME_SRCS src/runtime/hexagon/hexagon_posix.cc)
```

Comment:
Because `posix_memalign` function is present in static libc (so you'd get multiple definitions when building a static libtvm_runtime.a). When linking a shared library (libtvm_runtime.so), libc is not linked in, and `posix_memalign` is not always present in the runtime system.

Reply:
could you comment this into the code?

Reply:
Done.